tests/builders/test_builders.py

```diff
@@ -138,3 +138,15 @@ def test_file_count():
         configuration=config,
     )
     cmd.build()
+
+def test_file_regex():
+    """This test buildspec using status check with 'regex_on_filename'"""
```

Comment:
So this docstring is not correct. The filename is `regex_on_filename` but that not what the test is doing. The test is performing status check on keyword `file_regex`. See https://buildtest.readthedocs.io/en/devel/buildspecs/buildspec_overview.html#specify-regular-expression-on-arbitrary-filenames

I would rephrase this docstring to 

```python
def test_file_regex():
    """This test will perform status check with regular expression on file via 'file_regex'"""
```

Reply:
@shahzebsiddiqui, the filename is regex_on_filename and the test is performing on regex_on_filename keyword. She got it and I commented "The comment in the methods matches the linked yml file" 14 hours ago.

Reply:
Yes what you said is correct but the docstring makes no sense it is too technical and tied to the name of file and test. The actual keyword that is being tested is `file_regex` which is what is used in buildspec to specify a filename to perform regular expression.

Reply:
Okay, thanks,
@braeeast2001 please see the above comment by @shahzebsiddiqui and update your docstring.